themes/src/main/resources/theme/base/account/deleteAccount.ftl

```diff
@@ -0,0 +1,48 @@
+<#import "template.ftl" as layout>
+<@layout.mainLayout active='deleteAccount' bodyClass='deleteAccount'; section>
+
+    <div class="row">
+        <div class="col-md-10">
+            <h2>${msg("deleteAccount")}</h2>
+        </div>
+    </div>
+
+
+    <form action="${url.deleteAccountUrl}" class="form-horizontal" method="post">
+
+       <div class="alert alert-warning" style="margin-top:0 !important;margin-bottom:30px !important">
+           <span class="pficon pficon-warning-triangle-o"></span>
+           ${msg("irreversibleAction")}
+       </div>
+
+        <div class="row">
+          <div class="col-md-10">
+                <p>${msg("deletingImplies")}<p>
+                <ul>
+                  <li>${msg("loggingOutImmediately")}</li>
+                  <li>${msg("errasingData")}</li>
+                  <li>${msg("accountUnusable")}</li>
+                </ul>
+          </div>
+        </div>
+
+         <div class="row">
+            <div class="col-md-10">
+               ${msg("initialDeletionInstruction")}
+            </div>
+         </div>
+
+        <input type="text" id="username" name="username" value="${(account.username!'')}" autocomplete="username" readonly="readonly" style="display:none;">
```

Comment:
Why to send the username?

Reply:
Seems to be not needed. Going to potentially remove this file, as this is old console